rosbag2_cpp/src/rosbag2_cpp/writers/sequential_writer.cpp

```diff
@@ -110,6 +110,7 @@ void SequentialWriter::open(
   }
 
   bool dir_created = rcpputils::fs::create_directories(db_path);
+  dir_created &= db_path.is_directory();
```

Comment:
This stands in for https://github.com/ros2/rcpputils/pull/98/files (maybe it should be backported to Foxy?)